flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorITCase.java

```diff
@@ -174,70 +151,18 @@ protected Cluster buildCluster(Cluster.Builder builder) {
         }
     }
 
-    private static class EmbeddedCassandraService {
-        CassandraDaemon cassandraDaemon;
-
-        public void start() throws IOException {
-            this.cassandraDaemon = new CassandraDaemon();
-            this.cassandraDaemon.init(null);
-            this.cassandraDaemon.start();
-        }
-
-        public void stop() {
-            this.cassandraDaemon.stop();
-        }
+    public static CassandraContainer createCassandraContainer(String imageTag) {
+        CassandraContainer cassandra =
+                new CassandraContainer(DockerImageName.parse("cassandra").withTag(imageTag));
```

Comment:
the cassandra image must be defined in `org.apache.flink.util.DockerImageVersions` for the caching to work.

Reply:
Ok thanks I did not know that

Reply:
That's not surprising isn't it.

It's certainly not ideal, but we need a place to define the docker images + versions so that we can use them as a cache key on CI. Otherwise, you could bump the version in the test, but CI would continue to use the old outdated cache.

Reply:
makes total sense